js/sider/OppfolgingsplanSide.js

```diff
@@ -177,7 +163,7 @@ export class OppfolgingsplanSide extends Component {
             return <AppSpinner />;
           } else if (hentingFeilet || sendingFeilet) {
             return <Feilmelding />;
-          } else if (!tilgang.data.harTilgang || !sykmeldt || !harSykmeldtGyldigSykmelding) {
+          } else if (!tilgang.data.harTilgang || !sykmeldt) {
```

Comment:
Hva er tanken bak at man har fjernet denne sjekken? Er det slik at man skal kunne se oppfølgingsplan for sykmeldinger eldre enn 4 måneder?

Reply:
Jeg antar jo at det er bevisst at man har fjernet denne sjekken, og i så fall bør man rydde opp litt ubrukt kode:
- I oppfolgingsplanUtils.js er det nå en ubrukt funksjon:  sykmeldtHarGyldigSykmelding 
- Den ovennevnte bruker erSykmeldingGyldigForOppfolgingMedGrensedato som ikke lenger er brukt heller

Reply:
Når jeg tenker meg om så er det vel kanskje slik at vi kan rydde opp alle referanser til sykmeldinger dersom denne sjekken ikke lenger skal brukes.

Reply:
Det nye endepunktet vi bruker for å hente sykmeldt har denne sjekken hos seg, så da trenger vi ikke hente sykmeldinger hos oss for å sjekke 4-mnd regelen :) Men ja, kan rydde opp i sykmeldinger-referanser. Fikser

Reply:
(ellers hørte jeg fra Erik at det har blitt diskutert tidligere å fjerne 4-mnd sjekken for oppfølgingsplaner, slik at man kan gå inn uansett)